docs/sources/developers/plugins/add-support-for-annotations.md

```diff
@@ -8,140 +8,26 @@ This guide explains how to add support for [annotations]({{< relref "../../dashb
 
 This guide assumes that you're already familiar with how to [Build a data source plugin]({{< relref "/tutorials/build-a-data-source-plugin.md" >}}).
 
-Data sources in Grafana can support [Annotations]({{< relref "../../dashboards/annotations.md" >}}) by handling _annotation queries_.
-
-Handling annotation queries is similar to how you'd handle a metrics query. The difference is that instead of returning [data frames]({{< relref "data-frames.md" >}}), an annotation query returns _annotation events_.
+> **Note:** Annotation support for React plugins was released in Grafana 7.2. To support earlier versions, refer to the [Add support for annotation for Grafana 7.1](https://grafana.com/docs/grafana/v7.1/developers/plugins/add-support-for-annotations/).
 
 ## Add annotations support to your data source
 
-To add support for annotations to an existing data source, you need to:
-
-- Enable annotations support
-- Override the `annotationQuery` method
-- Construct annotation events
-
-### Enable annotations support
-
-Tell Grafana that your data source plugin can return annotations events by adding `"annotations": true` to the [plugin.json]({{< relref "metadata.md" >}}) file.
-
-```json
-{
-  "annotations": true
-}
-```
-
-### Override the `annotationQuery` method
-
-In `DataSource.ts`, override the `annotationQuery` method from `DataSourceApi`.
-
-```ts
-async annotationQuery(options: AnnotationQueryRequest<MyQuery>): Promise<AnnotationEvent[]> {
-  return [];
-}
-```
-
-### Construct annotation events
-
-Return an array of [AnnotationEvent]({{< relref "../../packages_api/data/annotationevent.md" >}}).
-
-```ts
-async annotationQuery(options: AnnotationQueryRequest<MyQuery>): Promise<AnnotationEvent[]> {
-  const events: AnnotationEvent[] = [];
-
-  const date = new Date();
-
-  const event: AnnotationEvent = {
-    time: date.valueOf(),
-    text: 'foo',
-    tags: ['bar'],
-  };
-
-  events.push(event);
-
-  return events;
-}
-```
-
-## Region annotations
-
-[Region annotations]({{< relref "../../dashboards/annotations.md#adding-regions-events" >}}) have a start and end time. This can for example be used to annotate maintenance windows or downtime.
-
-To return a region annotation, set the `timeEnd`, and `isRegion` properties.
+To enable annotation support for your data source, add the following two lines of code. Grafana uses your default query editor for editing annotation queries.
```

Comment:
Nitpick: but personally I would like a bit more information and context, something like "add `annotation: true` to the plugins capabilities list so Grafana knows it supports annotations see plugin json config docs for details."